canonical/ubuntu.com
# Rebrand /landscape

## Done

- Rebranded /landscape landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [Demo](https://ubuntu-com-14540.demos.haus/landscape)
- [Design](https://www.figma.com/design/btsxPLFKGPWDlbQ4rjR4jZ/25.04-Landscape-Rebranding?node-id=0-1&node-type=canvas&m=dev)
- [Copydoc](https://docs.google.com/document/d/1Q5KPVOn3yaexgSbkZHmXHmWzT5WG3ocYAPthweMwDz4/edit?tab=t.0)
## Issue / Card

Fixes # [WD-16834](https://warthogs.atlassian.net/browse/WD-16834)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-16834]: https://warthogs.atlassian.net/browse/WD-16834?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
